### ISSUE TITLE
RuntimeError:Given groups=1

### ISSUE THREAD

**Issue by kungkook** [jfzhang95/pytorch-deeplab-xception#160]
An error occurred while I was trying to train the VOC2007 data set:
RuntimeError:Given groups=1, weight of size 256 304 3 3,expected input[1,2096,129,129]to have 304 channels, but got 2096 channels instead.
The error code is:

self.last_conv = nn.Sequential(nn.Conv2d(304, 256, kernel_size=3, stride=1, padding=1, bias=False),
BatchNorm(256),
nn.ReLU(),
nn.Dropout(0.5),
nn.Conv2d(256, 256, kernel_size=3, stride=1, padding=1, bias=False),
BatchNorm(256),
nn.ReLU(),
nn.Dropout(0.1),
nn.Conv2d(256, num_classes, kernel_size=1, stride=1))
........
x = self.last_conv(x)

Could you tell me how to get the input channel 304?
